Formula/dumb.rb

```diff
@@ -0,0 +1,18 @@
+class Dumb < Formula
+  desc "IT, XM, S3M and MOD player library"
+  homepage "http://dumb.sourceforge.net/index.php?page=about"
+  url "https://downloads.sourceforge.net/project/dumb/dumb/0.9.3/dumb-0.9.3.tar.gz"
+  sha256 "8d44fbc9e57f3bac9f761c3b12ce102d47d717f0dd846657fb988e0bb5d1ea33"
+
+  def install
+    (buildpath/"make/config.txt").write <<-EOS.undent
+include make/unix.inc
+ALL_TARGETS := core core-examples core-headers
+PREFIX := #{prefix}
+    EOS
+    system "make"
+    bin.install ["examples/dumb2wav", "examples/dumbout"]
+    include.install ["include/dumb.h", "include/aldumb.h"]
+    lib.install ["lib/unix/libdumb.a", "lib/unix/libdumbd.a"]
+  end
```

Comment:
That should be submitted as an upstream bug report.

For the formula, could it be done more simply with a couple `mkdir`s and then `make install-core-examples` and `make install-core-headers`?